physionet-django/console/templates/console/credential_processing.html

```diff
@@ -57,7 +57,7 @@
                 <td>{{ user.email }}</td>
                 <td>{{ application.reference_email }}</td>
                 <td>{{ application.application_datetime|date }}</td>
-                <td class="{{application.get_trafficlight_status}}">{{""}}</td>
+                <td><i class="traffic-circle traffic-{{application.get_traffic_status}}"></i></td>
```

Comment:
Nitpick, but please could you add the space on either side of the variable name? `{{ application.get_traffic_status }}`. See: https://docs.djangoproject.com/en/4.1/topics/templates/#variables